schema.sql

```diff
@@ -10,4 +10,33 @@
 
 -- for the second day's project
```

Comment:
- [ ] Please all animals should got assigned owners , as you know we changed the owners to use the owners_id so your code should set the owner_id for the users that own the animals